src/pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -73,7 +75,7 @@ def resume_start(self, checkpoint_path: Optional[_PATH] = None) -> None:
         3. from `checkpoint_path` file if provided
         4. don't restore
         """
-        self.resume_checkpoint_path = checkpoint_path
```

Comment:
Let's keep this. It communicates that this is the checkpoint path to resume from, not to be confused with the one being saved. It gets managed by `resume_start()` and `resume_end()`

Reply:
This is still there, but renamed to `_ckpt_path`. Are you suggesting to make it public? Or to keep the old name?

I would like that this name matches the one in the trainer, so that would mean having `trainer.resume_checkpoint_path` which then would be inconsistent with `trainer.test(ckpt_path)|

Reply:
> Are you suggesting to make it public? Or to keep the old name? 

Both. The existing attribute already sort of serves the same purpose, it is why it was public. We made `resume_checkpoint_path` so that it can be changed if needed.